src/content/release-notes.mdx

```diff
@@ -5,6 +5,38 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.26.0
+
+11 November 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- **Hostname Length Limit**: Deploys now fail if a service hostname exceeds 63 characters, showing an error message. Previously, the dev environment deployed successfully even if the endpoint didn’t work. This may affect environments that deployed without issues before.
+- **Helm Release Name Limit**: Helm release names are now limited in length. Use defaultBackend.nameOverride to shorten the default backend service name if needed. If changing this during an upgrade, [follow the guide as it may impact the installation](self-hosted/helm-configuration.mdx#manual-migration-steps-when-renaming-the-defaultbackend-service).
+- **Private Repository Deploys**: Deploying private repositories now uses the Okteto backend as the SSH agent, rather than mounting the local SSH agent. This brings feature parity between remote and local deploys but may affect repositories cloned within Okteto Manifest commands.
+- **Buildkit Persistence Enabled**: Buildkit persistence is now enabled by default, with a 100Gi disk and cache set to 90% of the disk size. If you previously used buildkit.persistence.cache, adjust to the new ratio, as this setting is no longer applicable.
+
+### New Features
+
+- **Introducing the [Okteto Resource Manager](admin/resource-manager.mdx)**: a new feature that automatically optimizes CPU and memory requests for your environments. By analyzing real-time resource utilization, the Resource Manager dynamically adjusts resource requests to ensure efficient usage, prevent node overload, and improve overall cluster performance. This feature simplifies resource management, reduces manual adjustments, and enhances application stability, especially in larger clusters
+- Added Okteto [Garbage Collector settings to the Admin Dashboard](admin/cleanup.mdx): Admins of Okteto can now manage sleep and delete periods from the Dashboard
+- Added additional feature and documentation for [running Buildkit at scale](self-hosted/manage/buildkit-high-performance.mdx)  <!-- 8727 -->
+
+### Improvements
```

Comment:
As part of the CLI changes, we have added the ability of setting `context` field in the Okteto Manifest for `deploy` and `destroy` section to be used as the working directory of the commands specified on those sections. Something similar to the same field in the `test` section: https://www.okteto.com/docs/1.26/reference/okteto-manifest/#context-string-optional

We should probably add documentation about that field, I think it wasn't added.

Another improvement we have added in the CLI is the retry on build operations if BuildKit is not available. CLI will automatically retry builds if there is a transient error connecting with BuildKit. We have added the following env vars to control this behavior: `OKTETO_BUILDKIT_MAX_RETRIES_FOR_TRANSIENT_ERRORS`, `OKTETO_BUILDKIT_WAIT_TIMEOUT` and `OKTETO_BUILDKIT_RETRY_INTERVAL` . We can add a link to this page: https://www.okteto.com/docs/1.26/reference/feature-flags/